sycl/include/CL/sycl/detail/pi.h

```diff
@@ -365,6 +365,13 @@ typedef enum {
   PI_EVENT_INFO_CONTEXT = CL_EVENT_CONTEXT
 } _pi_event_info;
 
+typedef enum {
+  PI_PROFILING_COMMAND_QUEUED = CL_PROFILING_COMMAND_QUEUED,
```

Comment:
Please repeat enum type name as a prefix of enum values: PI_PROFILING_INFO_COMMAND_QUEUED

Reply:
Makes sense. It has been changed.